apps/hosts/views.py

```diff
@@ -212,20 +239,63 @@ def remove_self_from_challenge_host_team(request, challenge_host_team_pk):
 @api_view(['POST'])
 @permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
 @authentication_classes((ExpiringTokenAuthentication,))
-def invite_host_to_team(request, pk):
-
+def email_invite_host_to_team(request, pk):
```

Comment:
Please add docstring.

Reply:
I think you are missing bunch of conditions in this method. Let's say the new member is already a participant for a challenge which is hosted by the challenge host team where he is invited then he should not be able to join the team.

Reply:
@deshraj I've added it.